community-membership.md

```diff
@@ -219,3 +219,33 @@ The following apply to the subproject for which one would be an owner.
 - Ensure a healthy process for discussion and decision making is in place.
 - Work with other maintainers to maintain the project's overall health and
   success holistically.
+
+### Resolving technical conflicts within a SIG
```

Comment:
should it be h4 (subsection of maintainer's "Responsibilities and privileges")? Or this applied wider, also to approvers?

Reply:
I think this applies specifically to Maintainers (the next-highest H2), and I don't think of this as a Maintainer responsibility as much as "something relevant to maintainers"... I don't feel strongly and can change this if you do. :)

Reply:
ok, this will do than